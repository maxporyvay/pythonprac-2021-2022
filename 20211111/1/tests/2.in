A, B = OrderedComplex(1, 8), OrderedComplexMul(-4, 11)
print(A<B, B@B)
