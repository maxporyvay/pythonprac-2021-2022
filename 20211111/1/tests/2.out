False 137.0
